machine_configuration/README.md

```diff
@@ -87,6 +87,18 @@ $ lsmod | grep ast
 ```
 Ensure that ast driver is not loaded.
 
+# Machine Configuration for Provisioning Intel QAT
+
+* Turn on `intel_iommu` kernel parameter for QAT and load `vfio-pci` at boot
+
+For QAT, We need to turn on `intel_iommu` kernel parameter. Also we need to load `vfio-pci` before we can deploy the QAT plugin. Both can be done my same MCO using the following command. 
```

Comment:
I think we can remove line 94.

Reply:
both sentences and just start with command?

Reply:
both sentences please.
Line 92 is good enough.